hardhat/hardhat.multiplewinners.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @name chainid
+ */
+ task("prizeSplits")
+ .addPositionalParam("address", "Contract address")
+ .setAction(async function ({ address }) {
+  const contract = await ethers.getContractAt("MultipleWinners", address);
+  const prizeSplits = await contract.prizeSplits();
+  console.log(prizeSplits, 'prizeSplits')
+
+  return prizeSplits
+});
+
+task("set-prizeSplits")
+.addPositionalParam("address", "Contract address")
+.setAction(async function ({ address }) {
+ const contract = await ethers.getContractAt("MultipleWinners", address);
+
+ const firstPrizeSplitConfigs = [
+  {
+    target: '0x05dB4BE3A9D6F08230254EC9693Bbef0C9de3858',
+    percentage: "100",
+    token: 0,
+  },
+];
+
+ const prizeSplits = await contract.setPrizeSplit(firstPrizeSplitConfigs);
```

Comment:
You need to pass an `index` in last param no?

Reply:
No index is required. I think you might be reviewing an older PR for this.

Reply:
This will ultimately be deleted. Was simply using for debugging on previous version.